.circleci/config.yml

```diff
@@ -70,11 +87,16 @@ commands:
       - run:
           name: "Build packages"
           command: npm run build
-  cmd_run-e2e-tests:
+  cmd_run-e2e-tests-unix:
     steps:
       - run:
           name: "Run e2e tests"
           command: cd packages/flex-plugin-e2e-tests && npm start
+  cmd_run-e2e-tests-win:
+    steps:
+      - run:
+          name: "Run e2e tests"
+          command: cd packages/flex-plugin-e2e-tests; npm start
```

Comment:
Seriously? That's why these two are different? Windows back doesn't understand `&&`?

Reply:
What if you change this command to 
```
command: |
  cd packages/flex-plugin-e2e-tests
  npm start
```
then use the exact same command for both?

Reply:
I can give that a go - but I was looking for a way which works for both bash and powershell. Later versions of powershell do support && but the CircleCI one doesn't. But you pointed out a different issue - I shouldn't be running it both against bash and powershell since we don't have different credentials per shell. Will leave it at bash and change the command for now.